packages/code/src/renderingPlugin.tsx

```diff
@@ -54,6 +54,18 @@ function isNumber(item: any): item is number {
   return item != null && typeof item === 'number';
 }
 
+function isTokenObject(item: any): item is TokenObject {
```

Comment:
should we type this as unknown rather than any?

Reply:
If that's typed as unknown, then TS thinks the `kind` and `children` properties of the object don't exist.

Reply:
perhaps `unknown | TokenObject`

Reply:
What's the end result you're hoping changing this type will achieve?

Reply:
Just being more precise with our typings that's all

Reply:
`unknown | TokenObject` has the same issue. Using any here is fine I think. I'd want to be more specific if it were a return type or something similar.